src/inmanta/ast/statements/call.py

```diff
@@ -238,6 +238,11 @@ def call_in_context(
             except UnknownException as e:
                 result.set_value(e.unknown, self.ast_node.location)
             except UnsetException as e:
+                call: str = "{module}::{func}".format(module=self.plugin.__module__, func=self.plugin.__function_name__)
+                location: str = "({file}:{line})".format(file=self.ast_node.location.file, line=self.ast_node.location.lnr)
```

Comment:
It might be better to do `str(location)` to get consistent formatting with other locations?